src/Chapter14/Listing14.01.HeaterAndCoolerEventSubscriberImplementations.cs

```diff
@@ -24,14 +26,15 @@ public void OnTemperatureChanged(float newTemperature)
         }
     }
 
-    class Heater
+    public class Heater
     {
         public Heater(float temperature)
         {
             Temperature = temperature;
         }
 
-        // Cooler is activated when ambient temperature is higher than this
+        // Cooler is activated when ambient temperature
+        // is higher than this
```

Comment:
```suggestion
        // is lower than this
```
Since it's a heater would it activate when the temperature is lower than the goal?